test/sanity/validate-modules/ignore.txt

```diff
@@ -1207,7 +1207,6 @@ lib/ansible/modules/storage/netapp/na_cdot_volume.py E322
 lib/ansible/modules/storage/netapp/na_cdot_volume.py E324
 lib/ansible/modules/storage/netapp/na_cdot_volume.py E325
 lib/ansible/modules/storage/netapp/netapp_e_amg.py E322
```

Comment:
I think this change needs removing from this PR (as it's been done in another PR which has since been merged)

Also you need to rebase to resolve the merge conflict https://docs.ansible.com/ansible/devel/dev_guide/developing_rebasing.html